spec/dummy/db/schema.rb

```diff
@@ -43,6 +43,12 @@
 
   add_index "addresses", ["registration_id"], name: "index_addresses_on_registration_id", using: :btree
 
+  create_table "defra_ruby_exporters_bulk_export_files", force: :cascade do |t|
+    t.string   "file_name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
I don't think this should be part of this PR :)

Reply:
I was being lazy but you are right, should be in a separate one. Removed in https://github.com/DEFRA/waste-exemptions-engine/pull/138/commits/a0bc400746aecc57ff0119154702293a08ca3836